traffic_ops/app/lib/API/User.pm

```diff
@@ -277,6 +285,7 @@ sub current {
 			@data, {
 				"id"              => "0",
 				"username"        => $current_username,
+				"tenantId"	  => $self->current_user_tenant(),
```

Comment:
Thanks:)
Indeed we plan to "seed" an active "root" tenant and potentially an admin user with this tenancy (I still need to understand how currently, without tenancy, the first admin user is added, and maybe follow the same path). 
Note however that in the first phase we allow the system to work with no tenancy set and until tenancy is in use the LDAP users tenancy should be "undef". 
Beyond the fact that automatically identifying the point in time a the feature is started to be in use may not be simple, I'm not comfortable with a solution in which the tenancy of the LDAP users is changed at some point in time without the operator implicitly set it. It sounds like a change in behavior only "experts" would understand.

Nevertheless, as a LDAP user has a read-only capabilities, may we consider the below solution?
current_user_tenant keeps returning "undef" for LDAP users. However, when we test what a user is allowed to do (future PR), if the user is an LDAP user he can "read" everything.